core/dbt/parser/models.py

```diff
@@ -108,13 +112,37 @@ def _get_call_literals(self, node):
         return arg_literals, kwarg_literals
 
     def visit_Call(self, node: ast.Call) -> None:
-
-        func_name = self._flatten_attr(node.func)
-        if func_name in ["dbt.ref", "dbt.source", "dbt.config", "dbt.config.get"]:
-            # drop the dot-dbt prefix
-            func_name = func_name.split(".")[-1]
-            args, kwargs = self._get_call_literals(node)
-            self.dbt_function_calls.append((func_name, args, kwargs))
+        if isinstance(node.func, ast.Attribute) and node.func.attr in dbt_function_key_words:
```

Comment:
@gshank  Do you have any experience with language parser? feels like we might be re-inventing the wheels now